test/card_test.rb

```diff
@@ -4,20 +4,16 @@
 
 class CardTest < Minitest::Test
   def setup
-    card = Card.new(:diamond, 'Queen', 12)
+    @card = Card.new(:diamond, 'Queen', 12)
   end
 
   def test_it_exists
-    card = Card.new(:diamond, 'Queen', 12)
```

Comment:
Removed creation of new Card objects because the def setup was changed to instances.